packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.tsx

```diff
@@ -66,12 +66,7 @@ export class GuxVisualization {
         visSpec.signals.push({
           name: 'chartClick',
           value: 0,
-          on: [
-            { events: 'rect:mousedown', update: 'datum' },
-            { events: 'arc:mousedown', update: 'datum' },
-            { events: 'line:mousedown', update: 'datum' },
-            { events: 'symbol:mousedown', update: 'datum' }
-          ]
+          on: [{ events: '*:mousedown', update: 'datum' }]
```

Comment:
I noticed that `gux-chart-column` logs when the chart is clicked with this line `chart1.addEventListener('chartClicked', (event) => console.info(event));` - do you think it's worth adding in `gux-visualization` to demonstrate this chart click event firing properly on the example page?

Reply:
is that different than the log that already shows on the example page when you click?

Reply:
I'm not seeing a log on the example page for `gux-visualization`, but I do see a log shows on the example page for `gux-chart-column`

Reply:
Oh I mean the notification. Not sure if it's worth having the log if we already show a notification for the chartClicked event

Reply:
Agreed. I didn't notice the notification at first, but that's good as is.